app/controllers/employee_invitation_controller.rb

```diff
@@ -3,14 +3,29 @@ class EmployeeInvitationController < ApplicationController
 
   def create
     @email = params[:email]
-    @user = User.find_by(email: @email)
+    @user = current_user
 
-    message, status = CheckEmployeeAssociation.call(@user, @email, @company)
+    message, status = InviteEmployeeService.call(@user, @email, @company)
       .slice(:message, :status).values
 
     render json: { message: message }, status: status
   end
 
+  def update
+    @token = params[:token]
+    @accepted = params[:accepted]
+
+    if (invite = Invite.available.find_by(invitation_token: @token))
+      case @accepted
+      when true
+        return render json: { message: 'invite accepted' }, status: :ok if invite.accept
+      when false
+        return render json: { message: 'invite refused' }, status: :ok if invite.refuse
+      end
+    end
+    render json: { message: 'invite could not replied' }, status: :unprocessable_entity
```

Comment:
Does it make sense?
```suggestion
    # it will render 404 if invite not found, more info:
    # https://github.com/comarev/comarev/blob/98a0c2cb43b673ca5a6948a6ca4f0949ac4f1fbc/app/controllers/concerns/error_rescuable.rb#L5
    invite = Invite.available.find_by!(invitation_token: @token) 
   
    if @accepted
      invite.accept && render json: { message: 'invite accepted' }
    else
       invite.refuse && render json: { message: 'invite refused' }
    end
```